RPIPs/RPIP-30.md

```diff
@@ -0,0 +1,123 @@
+discussions-to: https://dao.rocketpool.net/t/rpl-staking-rework-proposal/2090
+status: Draft
+type: Protocol
+category: Core
+created: 2023-08-11
+---
+
+## Abstract
+RPL inflation can be seen as protocol revenue that we steer to various places to achieve protocol
+goals. This proposal looks to better align spending with protocol goals, specifically with the
+portion of inflation (70%) that goes to Node Operators today.
+
+**The primary goal of the proposal is to increase spend aligned with rETH supply creation.** This
+means both rewarding based on minipool count and rewarding LEB8s over LEB16s (as they create more
+rETH supply). Mechanically speaking, this means rewarding the minimum and near-minimum staked RPL
+more, and basing rewards on borrowed ETH.
+
+Secondarily, the proposal also aims to improve quality of life by avoiding locking people in far
+from their desired asset allocation (which may lead to churn as folks exit to rebalance), retain
+enough friction to avoid adding market volatility, and phase the new rules in slowly to minimize
+market impacts.
+
+## Rationale
+**Please** see the proposal document on
+[Valdorff's github](https://github.com/Valdorff/rp-thoughts/tree/main/rpl_staking), or its backup
+printout [here](../assets/rpip-draft/rpl_staking_readme.pdf). It greatly expands on the context, 
+reasoning, and impacts for this proposal.
+
+As a very high level summary, the below pie charts show overall intended spend before and after this
+proposal. Mechanically, the blue categories are RPL staked from 10-15% of borrowed ETH value
+(incentivizing), RPL staked from 15-30% of borrowed ETH value (overshoot), and RPL staked above 30%
+of borrowed ETH value (speculation).
+
+| ![image](../assets/rpip-draft/overall_spend_pie_curr.png) | ![image](../assets/rpip-draft/overall_spend_pie_prop.png) |
+|:---------------------------------------------------------:|:-------------------------------------:|
+
+The proposal focuses a lot more spend on incentivizing minipool creation, while still spending a
+significant amount on the speculative and hands-off categories of RPL staking (compare with Dev
+spend, for example). The current outsized speculation category (larger than Dev, oDAO, IMC, GMC,
+and Reserves combined) has been significantly reduced so that we can spend a lot more on achieving
+the protocol goal of attracting minipool creation (and rETH supplying).
+
+## Specification
+
+Definition: X=1 is the first reward snapshot after (a) the end of the vote and (b) the
+implementation of the new rewards rules. X=2 is the snapshot after that, etc.
+
+- The new rewards rules SHALL be phased in such that, for reward snapshot X:
+  - For periods X=1 to X=5: a nodes' share of rewards is
+    `(X/6)*proposed_method_share + ((6-x)/6)*current_method_share`
+  - For periods X>5, they SHALL  simply be `proposed_method_share`
+  - `current_method_share` SHALL determine share of NO rewards among NOs according to the latest tree
```

Comment:
so this RPIP prevents any changes to the treegen spec for 5 periods?

Reply:
That was not the intent, but I can see how that would be read that way. I'll add an explicit "MAY" be changed following existing processes for both current and proposed methods.